### PR TITLE
Add tests and release jobs into drone

### PR DESCRIPTION
**Pull request by LukasHirt**
Steps that should happen:
- Tests
	- Lint
	- Build
- Release (on tag)
	- Build
	- Github release
	- NPM release

### CODE REVIEW

**Comment by LukasHirt** [comment]
This PR is built on top of #2 After it gets merged I'll do a rebase.

**Comment by LukasHirt** [comment]
Had to change the node images because version 12 is having troubles with the module type in package.json (see https://github.com/manuelbieh/geolib/issues/208)

File: .drone.star [hunk 1]
```diff
@@ -219,4 +224,112 @@ def notify():
...
+      {
+        'name': 'publish-github',
+        'image': 'plugins/github-release:latest',
+        'pull': 'always',
+        'settings': {
+          'api_key': {
+            'from_secret': 'github_token',
+          },
+          'title': ctx.build.ref.replace("refs/tags/v", ""),
+          'note': 'dist/CHANGELOG.md',
```
Comment (micbar) [inline .drone.star:301]: You need to build the changelog first. The other alrready built changelog is from another pipeline. Also  you need only the changes for that specific tag. Example in https://github.com/owncloud/ocis/blob/master/.drone.star#L565

Comment (LukasHirt) [reply .drone.star:319]: I see, thanks! I added it.

**Comment by LukasHirt** [comment]
Forgot to add changelog into makefile. Fixed in latest commit.

**Review by micbar — APPROVED** [review]

**Pull request merged**